src/components/expansion-panel/expansion-panel.ts

```diff
@@ -0,0 +1,235 @@
+import { LitElement, html } from 'lit';
+import { property, query } from 'lit/decorators.js';
+import { Constructor } from '../common/mixins/constructor.js';
+import { EventEmitterMixin } from '../common/mixins/event-emitter.js';
+import { themes } from '../../theming/theming-decorator.js';
+import { styles } from './themes/light/expansion-panel.base.css.js';
+import { styles as bootstrap } from './themes/light/expansion-panel.bootstrap.css.js';
+import { styles as fluent } from './themes/light/expansion-panel.fluent.css.js';
+import { styles as indigo } from './themes/light/expansion-panel.indigo.css.js';
+
+let NEXT_ID = 0;
+export interface IgcExpansionPanelComponentEventMap {
+  igcOpening: CustomEvent<any>;
+  igcOpened: CustomEvent<any>;
+  igcClosing: CustomEvent<any>;
+  igcClosed: CustomEvent<any>;
```

Comment:
Please provide the correct type for the custom event `detail` property, since it is not `any`.